administrator/components/com_media/Controller/Api.php

```diff
@@ -104,7 +105,20 @@ public function files()
 			switch ($method)
 			{
 				case 'get':
-					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'));
+					$option = $this->input->getWord('opt');
```

Comment:
For example the google calendar api has a flag `alwaysIncludeEmail` which returns the email addresses in the response or not https://developers.google.com/google-apps/calendar/v3/reference/events/list. I would rather use something like that then returning a totally different response on a GET request for the same endpoint.